src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -191,73 +184,35 @@ private void testCreateInstructorAction() throws Exception {
-
-        ______TS("new instructor can edit feedbackSession of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        InstructorFeedbackEditPage feedbackEditPage =
-                instructorHomePage.clickFeedbackSessionEditLink(demoCourseId, "Second team feedback session");
-
-        feedbackEditPage.clickEditSessionButton();
-
-        FeedbackSessionAttributes feedbackSession =
-                getFeedbackSessionWithRetry(demoCourseId, "Second team feedback session");
-        feedbackEditPage.editFeedbackSession(feedbackSession.getStartTime(),
-                                             feedbackSession.getEndTime(),
-                                             new Text("updated instructions"),
-                                             feedbackSession.getGracePeriod());
-        feedbackEditPage.reloadPage();
-        instructorHomePage.verifyHtmlMainContentWithReloadRetry("/newlyJoinedInstructorFeedbackSessionSuccessEdited.html");
-
-        ______TS("new instructor can click submit button of sample feedbackSession");
-        instructorHomePage.loadInstructorHomeTab();
-        FeedbackSubmitPage fbsp = instructorHomePage.clickFeedbackSessionSubmitLink(demoCourseId,
-                                                                                    "Second team feedback session");
-        fbsp.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSubmissionEditPage.html");
-
-        ______TS("new instructor can send reminder of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        instructorHomePage.clickFeedbackSessionRemindLink(demoCourseId,
-                                                          "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSessionRemind.html");
-
-        ______TS("new instructor can unpublish feedbackSession of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        instructorHomePage.clickFeedbackSessionUnpublishLink(demoCourseId, "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSessionUnpublished.html");
-
-        ______TS("new instructor can publish feedbackSession of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        instructorHomePage.clickFeedbackSessionPublishLink(demoCourseId, "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSessionPublished.html");
-
-        feedbacksPage.logout();
+        resultsPage = instructorHomePage.clickFeedbackSessionViewResultsLink(
+                demoCourseId, "Session with different question types");
+        resultsPage.expandPanels();
+        resultsPage.verifyHtmlMainContent("/newlyJoinedInstructorThirdFeedbackSessionResultsPage.html");
 
+        //verify the given email is valid or not
         ______TS("action failure : invalid parameter");
```

Comment:
Instead of adding a comment, modify line 216 which is supposed to serve as a description of the test case.